tests/test_transforms/test_timestamp/test_fourier_transform.py

```diff
@@ -166,3 +167,17 @@ def test_forecast(ts_trend_seasonal):
 def test_save_load(ts_trend_seasonal):
     transform = FourierTransform(period=7, order=3)
     assert_transformation_equals_loaded_original(transform=transform, ts=ts_trend_seasonal)
+
+
+@pytest.mark.parametrize(
+    "transform",
```

Comment:
We should probably also test `FourierTransform(period=7, mods=[1])` but it will fail because during setting `mods=None` there will be no `order` or `mods` remaining.

Reply:
I added a new test below.